ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/InputsAndEffects.scala

```diff
@@ -44,7 +43,7 @@ private[kvutils] object InputsAndEffects {
     * and packages.
     */
   def computeInputs(
-      tx: Transaction.Transaction,
+      tx: CommittedTransaction,
```

Comment:
At the point where `computeInputs` is called, the transaction isn't committed yet. Correct, @miklos-da?

Reply:
`CommittedTransaction` does not mean the transaction is committed, but rather it has the format of a `CommittedTransaction`. 

The ledger is free to make this conversion a bit where it what.
* In sandbox classic this is done in the persistent layer just when writing the data
* In corda it is done when reading the data from the ledger
* In kvutil it is done `KeyValueParticipantStateWriter#submitTransaction` just before calling `computeInputs` (See original version in https://github.com/digital-asset/daml/blob/1ce67fbd2f7603fb1ab1346cca402248c65c3db1/ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/KeyValueParticipantStateWriter.scala#L33)

Reply:
For kvutil the name are a bit confusing right now, since there is no actual difference between Submitted and Commited Transaction. 
I am open to any other name for those Types. 
We could also completely remove the difference between this two type in kvutils and let it in the participant state. However It is not clear to me where is the boundary.